styleguide/_themes/global/scss/fonts.scss

```diff
@@ -13,10 +13,11 @@ $code: Tahoma, Arial, Geneva, Helvetica;
 @mixin subhead($color: $gray-darkest) {
   color: $color;
   font-family: $subhead;
-  font-weight: 400;
+  font-weight: 500;
+  letter-spacing: .35px;
   line-height: 1.45em;
   text-align: left;
-  text-transform: uppercase;
+  text-transform: capitalize;
```

Comment:
We should just remove text-transform here and use exactly what gets inserted into the CMS. There might be instances where ECMs want to use a product name that starts with lowercase.